docs/content/tutorial/step_15.ngdoc

```diff
@@ -0,0 +1,337 @@
+@ngdoc tutorial
+@name 15 - Accessibility
+@step 15
+@description
+
+<ul doc-tutorial-nav="15"></ul>
+
+In this step, we will learn how to fix the accessibility issues in our application.
+
+* We will fix some issues in the templates.
+* We will use the [protractor-accessibility-plugin](protractor-a11y-plugin) to review the accessibility of our application when we run our e2e tests.
+* We will explain how to add the {@link ngAria ngAria} module to use WAI-ARIA easily if you need it in your project.
+
+<div doc-tutorial-reset="15"></div>
+
+## Why accessibility matters
+
+A website or application accessible allows the users with different preferences and abilitities, to consume the content in a way that suits their needs.
+
+* [Creating an accessible web - Access iQ](https://www.youtube.com/watch?v=47n0wEcm6JU)
+* [Web Accessibility](https://www.youtube.com/watch?v=bEM9Fn9aOG8)
+* [Google Accessibility course](https://webaccessibility.withgoogle.com/course)
+* [Accessibility in AngularJS and Beyond](http://marcysutton.com/slides/fluent2015/)
+* [Introduction to Web Accessibility](https://www.w3.org/WAI/intro/accessibility.php)
+* [Apps For All: Coding Accessible Web Applications](https://shop.smashingmagazine.com/products/apps-for-all)
+
```

Comment:
Here's one more article that was written in response to Angular a11y: https://www.smashingmagazine.com/2015/05/client-rendered-accessibility/